ts/src/pro/probit.ts

```diff
@@ -36,15 +36,7 @@ export default class probit extends probitRest {
                     'filter': 'order_books_l2',
                     'interval': 100, // or 500
                 },
-                'watchTrades': {
-                    'filter': 'recent_trades',
-                },
-                'watchTicker': {
-                    'filter': 'ticker',
-                },
-                'watchOrders': {
-                    'channel': 'open_order',
-                },
+                'tempFiltersData': {}, // temporary container
```

Comment:
is this being used?

Reply:
@carlosmiei right, remnant. I've pushed all changes per your notes